### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by ahmedtaa**
# in this branch

- Implement a popup window that includes the project details on both desktop and mobile.
- Also the work grid is now loading dynamically with javascript.

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ahmedtaa 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️Your linters are properly set up and works fine.
- ✔️You have followed the correct GitHub flow
- ✔️Excellent commit messages.

 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

**I could not find your Readme in the files changed**
- [ ] **Required** Your readme file is missing a couple of headers such as the `Live Demo`, `Getting Started` etc. Kindly include them.
- [ ] **Required** Your screenshot is broken, kindly fix it. This is important to give the viewer a sample of what your project is.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -77,99 +127,7 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span>I'm a Software
         <div class="action-btn"><img src="img/Disabled.png" alt=""></div>
         <h2 class="headline"> My Recent Works </h2>
     </section>
```
Comment (brytebee) [inline index.html:129]: - [x] **Required** You are required to have a couple of projects in your website, that contain buttons to fire the popups when clicked, as the website is at the moment we do not have such projects. kindly ensure the projects are shown.
image
- [ ] **Required** The popups should also have X as a button to be clicked to close them.
- [ ] **Required** Finally, before requesting the next review, kindly go through the project requirement to ensure they are all met.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ahmedtaa 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️Your linters are properly set up and works fine.
- ✔️You have followed the correct GitHub flow
- ✔️Excellent commit messages.

 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

**I could not find your Readme in the files changed**
- [ ] **Required** Your readme file is missing a couple of headers such as the `Live Demo`, `Getting Started` etc. Kindly include them.
- [ ] **Required** Your screenshot is broken, kindly fix it. This is important to give the viewer a sample of what your project is.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

However, there are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

I have noticed that requested changes were not addressed, the last commits were updating the README file, 

- Please kindly follow the requirements of the previous reviewer and implement the project 🤗.
- Please add projects in your website, currently there are no projects to fire the popup window. At the end buttons and the modal should behave like this: 
chrome-capture

- Please make sure that everything works properly before requesting a review, because you have only 1 more chance to request a review. 

As a reference you can check [this](https://kandy-peter.github.io/My-portfolio/) project.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmedtaa** [comment]
Hi @shoirata, I do not clearly  understand the issue, because on my end everything is working  properly, I can open and close the module. Perhaps you reviewed another branch, please let me know. Thank you

**Comment by ShoiraTa** [comment]
Hi @ahmedtaa I have checked the repository and files, I am sorry, you are right, I checked the wrong branch. In your project everything is working properly. **No more valid requested changes.** The next reviewer  will approve the project. 
Dear reviewer, if something is wrong with the project, please contact @ahmedtaa  via slack or leave comments, he will reach out to you  🙏🏻

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @ahmedtaa ,  

Your project is complete! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**